spec/features/login/login_spec.rb

```diff
@@ -16,7 +16,7 @@
   fill_in :email, with: user.email
   fill_in :password, with: user.password
 
-  click_on "Submit"
+  find('#submit').click
```

Comment:
Did it take me an hour to figure out how to click the damn button? definitely.

Reply:
This is a good work around. I have definitely changed buttons to links in past projects, just because this was so frustrating.